config/application.rb

```diff
@@ -29,6 +29,10 @@ class Application < Rails::Application
       generate.system_tests false
     end
 
+    config.stripe.secret_key = sk_test_QK74QgtJV712dOkePnAST9Z3
```

Comment:
You never put an api key in version control. I want you to extract this to an ENV variable and put it in config/credentials.yml.enc, you need to open that file with `rails credentials:edit`.
Then you need to modify this line to `config.stripe.secret_key = ENV['stripe_secret_key']`